GameData/RealismOverhaul/Engine_Configs/Astris_Config.cfg

```diff
@@ -51,7 +51,7 @@
 		}
 		CONFIG
 		{
-			// source: http://www.b14643.de/Spacerockets/Diverse/European_Rocket_engines/engines.htm
```

Comment:
For the record, any link from b14643 that was originally under "Diverse" can be found under "Specials", meaning that the broken links just need a word replacement.